docs/en/Contributing-Getting-Started.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: global
+title: Contributor Getting Started
+nickname: Contributor Getting Started
+group: Resources
+---
+
+Welcome to the Alluxio community! We are excited for your contributions and engagement in the
+community! Here is a guide on how to get started with becoming a member in the Alluxio open source
+community.
+
+## Requirements
+
+The main requirement is a computer with Mac OSX or Linux installed. Alluxio does not have Windows support at the moment.
+
+## Software and Account Preparation
+
+There are several accounts and setup required before contributing to Alluxio.
```

Comment:
nit: grammar, maybe "There is some software and account setup required before contributing to Alluxio"